### PR TITLE
initial service setup for keyword/member autocomplete

### PR DESCRIPTION
**Pull request by sduval215**

### CODE REVIEW

File: src/stylesheets/scss/index.scss [hunk 1]
```diff
@@ -1,2 +1,3 @@
 @import 'components/App';
-@import './components/common';
\ No newline at end of file
+@import './components/common';
+@import './components/searchbox';
```
Comment (ravi-lattis) [inline src/stylesheets/scss/index.scss]: Add a empty last line pls.

File: src/stylesheets/scss/components/searchbox.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      font-size: 18px;
+      //border: 2px solid black;
+      cursor: pointer;
+      background: white;
+      &:hover{
+        background: #D8D8D8 !important;
+      }
+    }
+  }
+}
```
Comment (ravi-lattis) [inline src/stylesheets/scss/components/searchbox.scss]: Add a empty last line pls.

File: src/services/api.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    fetchKeyWords: function(){
+        return ['Gun Control', 'Gun Rights', 'Abortion', 'A.C.A', 'Campaign Reform'];
+    }
+}
```
Comment (ravi-lattis) [inline src/services/api.js]: Add a empty last line pls.

File: src/components/searchbox.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+                <ul id='search-options' className="no-padding no-margin text-left list-no-styles">
+                    <li>Paul Ryan</li>
+                    <li>Paul George</li>
+                </ul>
+            </div>
+        )
+    }
+}
+
+export default SearchBox;
```
Comment (ravi-lattis) [inline src/components/searchbox.js]: Add a empty last line pls.

Comment (ravirajsubramanian) [inline src/components/searchbox.js]: Add a empty last line pls.

File: src/services/api.js (hunk 3 above)
Comment (ravirajsubramanian) [inline src/services/api.js]: one more line.

File: src/stylesheets/scss/components/searchbox.scss (hunk 2 above)
Comment (ravirajsubramanian) [inline src/stylesheets/scss/components/searchbox.scss]: one more line.

File: src/stylesheets/scss/index.scss (hunk 1 above)
Comment (ravirajsubramanian) [inline src/stylesheets/scss/index.scss]: @sduval215 One more line here.

**Pull request merged**